machine_management/control_plane_machine_management/cpmso-about.adoc

```diff
@@ -25,7 +25,7 @@ The Control Plane Machine Set Operator has the following limitations:
 
 * The Operator requires the Machine API Operator to be operational and is therefore not supported on clusters with manually provisioned machines. When installing a {product-title} cluster with manually provisioned machines for a platform that creates an active generated `ControlPlaneMachineSet` custom resource (CR), you must remove the Kubernetes manifest files that define the control plane machine set as instructed in the installation process.
 
-* Only Amazon Web Services (AWS), Google Cloud Platform (GCP), Microsoft Azure, Nutanix, and VMware vSphere clusters are supported.
+* Only Amazon Web Services (AWS), Google Cloud Platform (GCP), {ibmpowerProductName} Virtual Server, Microsoft Azure, Nutanix, and VMware vSphere clusters are supported.
```

Comment:
Is there a product attribute for IBM Power that includes the registered trademark symbol? If so, please use it. According to the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#ibm-power), "On first use, mark "IBM" Power with a registered trademark icon (®)."

Reply:
@lahinson yes, attribute {ibmpowerProductName} is used for "IBM Power" in the code.